.github/workflows/ci.yml

```diff
@@ -578,7 +578,6 @@ jobs:
             - { modules: plugin/trino-postgresql }
             - { modules: plugin/trino-raptor-legacy }
             - { modules: plugin/trino-redis }
-            - { modules: plugin/trino-redshift }
```

Comment:
why? this means we no longer run things like `TestRedshiftConfig`, `TestRedshiftPlugin` etc.

Also why was this sneaked in through an unrelated PR?